src/ray/raylet/node_manager.cc

```diff
@@ -1496,23 +1496,22 @@ void NodeManager::HandleUpdateResourceUsage(
   for (const auto &resource_change_or_data : resource_usage_batch.batch()) {
     if (resource_change_or_data.has_data()) {
       const auto &resource_usage = resource_change_or_data.data();
-      const NodeID &node_id = NodeID::FromBinary(resource_usage.node_id());
-      if (node_id == self_node_id_) {
+      auto node_id = NodeID::FromBinary(resource_usage.node_id());
```

Comment:
all changes underneath is just refactoring right?

Reply:
just make it looks cleaner.